styx-common/src/main/java/com/spotify/styx/util/ReplayEvents.java

```diff
@@ -119,16 +120,10 @@ private ReplayEvents() {
     }
 
     RunState restoredState = RunState.fresh(workflowInstance, time);
-    try {
-      Map<WorkflowInstance, Long> instances = storage
-          .readActiveWorkflowInstances(workflowInstance.workflowId().componentId());
-      if (instances.keySet().contains(workflowInstance)) {
-        lastConsumedEvent = instances.get(workflowInstance);
-      } else {
-        lastConsumedEvent = sequenceEvents.last().counter();
-      }
-    } catch (IOException e) {
-      throw Throwables.propagate(e);
+    if (activeWorkflowInstances.keySet().contains(workflowInstance)) {
+      lastConsumedEvent = activeWorkflowInstances.get(workflowInstance);
```

Comment:
No biggie but could merge the `contains()` and `get()` into a single `get()` or `getOrDefault()`.